webui/react/src/services/decoder.ts

```diff
@@ -44,6 +44,16 @@ export const mapV1Permission = (permission: Sdk.V1Permission): types.Permission
   };
 };
 
+export const mapV1UserAssignment = (
+  assignment: Sdk.V1RoleAssignmentSummary,
+): types.UserAssignment => {
+  return {
+    isGlobal: assignment.isGlobal || false,
+    roleId: assignment.roleId || 0,
```

Comment:
We shouldn't use 0 as a special value. If this field is optional the type should be nullable.

Reply:
Backend did not do the annotation to require fields on their custom proto types for RBAC, only the request/response objects. Saw this before with Role. Just pushed a fix to this one.